### PR TITLE
Separate user page routes

### PR DESCRIPTION
**Pull request by msecret**
Fixes the loading state problems on the user page by having separate routes for org and space users and only loading the correct corresponding user types.

Also refactors the space router so each page has it's own function that all use a few central method for spaces.

### CODE REVIEW

File: static_src/components/main_container.jsx [hunk 1]
```diff
@@ -9,17 +9,23 @@ import Disclaimer from './disclaimer.jsx';
...
 import LoginStore from '../stores/login_store.js';
+import OrgStore from '../stores/org_store.js';
+import SpaceStore from '../stores/space_store.js';
 import { Nav } from './navbar.jsx';
 
-function getState() {
-  return { isLoggedIn: LoginStore.isLoggedIn() };
+function stateSetter() {
+  return {
+    currentOrgGuid: OrgStore.currentOrgGuid,
```
Comment (msecret) [inline static_src/components/main_container.jsx]: In another PR, I'll be moving to this pattern of always getting the guids from the stores rather then passing the initial one down for `main`. So components will consistently get these from the stores rather then from two places (props and stores). Is that an improvement?

Comment (jeremiak) [inline static_src/components/main_container.jsx]: Definitely an improvement. It also helps to keep the stores as the single source of truth about the state of the data. If we're going to have the stores keep track of what app/space/org is the "current" one, then the components should use the store to get that information.

**Comment by jeremiak** [comment]
This looks good to me! I like having a URL for each view

**Comment by msecret** [comment]
Ref #521

**Pull request merged**